ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -183,15 +206,23 @@ def create_cluster(cluster_name, ip, exclude_ports, base_dir, plugins=None):
         ports = System.get_free_ports(port_range, exclude_ports, 2, client)
         node_name = System.get_my_machine_id(client)
 
+        home_dir = ArakoonInstaller.ARAKOON_HOME_DIR.format(base_dir, cluster_name)
+        log_dir = ArakoonInstaller.ARAKOON_LOG_DIR.format(cluster_name)
+        tlog_dir = ArakoonInstaller.ARAKOON_TLOG_DIR.format(base_dir, cluster_name)
+
```

Comment:
It might be an idea to just create an ```ArakoonInstaller.ARAKOON_BASE_DIR``` containing ```{0}/arakoon/{1}``` so we can just rename that one to sth like ```{0}/arakoon/{1}-<timestamp>``` and move the contents of the (formatted) ```ArakoonInstaller.ARAKOON_LOG_DIR``` into that dir. That will save you calling the ```archive_existing_arakoon_data``` multiple times. You'll also be able to use that ```ArakoonInstaller.ARAKOON_BASE_DIR``` to remove the data on destroy.